heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -102,6 +102,7 @@ function process_message()
         msg.Fields.appBuildId       = app.buildId
         msg.Fields.vendor           = app.vendor
         msg.Fields.clientId         = parsed.clientId
+        msg.Fields.creationDate     = parsed.creationDate
 
```

Comment:
I was more thinking on the line that hopefully in the future, when we have a schema, we will extract more fields from the json blob and whoever does the analysis will expect those fields to match the schema. We could just rename it to creationDateTimestamp and adopt the same format as msg.Timestamp though.